src/com/layout/row/row.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component} 				from 'preact/preact';
+
+export default class Row extends Component {
+	constructor( props ) {
+		super(props);
+	}
+
+	render( props ) {
+		return (
+			<div {...props} class={cN([props.class, "-row"])}>
```

Comment:
can be simplified to `cn(props.class, "-row")`

Reply:
Not sure exactly what this compiling to, given that `{...props}` will also have a `props.class`.

Reply:
JSX should (and in practice for this particular instance at least) be overridden by the property if it's set again. So it's basically just forcing `-row` to be there.

Reply:
Ah shoot. I forgot to delete that comment. Clearly it's working. :wink: